code/modules/mob/living/carbon/human/species/station/protean_vr/protean_species.dm

```diff
@@ -0,0 +1,299 @@
+		BP_R_FOOT = list("path" = /obj/item/organ/external/foot/right/unbreakable/nano)
+		)
+
+	heat_discomfort_strings = list("You feel too warm.")
+	cold_discomfort_strings = list("You feel too cool.")
+
+	//These verbs are hidden, for hotkey use only
+	inherent_verbs = list(
+		/mob/living/carbon/human/proc/nano_regenerate, //These verbs are hidden so you can macro them,
+		/mob/living/carbon/human/proc/nano_partswap,
+		/mob/living/carbon/human/proc/nano_metalnom,
+		/mob/living/carbon/human/proc/nano_blobform,
+		/mob/living/carbon/human/proc/nano_change_fitting, //These verbs are displayed normally,
+		/mob/living/carbon/human/proc/shapeshifter_select_hair,
+		/mob/living/carbon/human/proc/shapeshifter_select_hair_colors,
+		/mob/living/carbon/human/proc/shapeshifter_select_colour,
+		/mob/living/carbon/human/proc/shapeshifter_select_eye_colour
+		)
+
+	var/global/list/abilities = list()
+
+	var/monochromatic = FALSE //IGNORE ME
+
+/datum/species/protean/New()
+	..()
+	if(!LAZYLEN(abilities))
+		var/list/powertypes = subtypesof(/obj/effect/protean_ability)
+		for(var/path in powertypes)
+			abilities += new path()
+
+/datum/species/protean/create_organs(var/mob/living/carbon/human/H)
+	var/obj/item/device/nif/saved_nif = H.nif
+	if(saved_nif)
+		H.nif.unimplant()
+		H.nif.forceMove(null)
+	..()
+	if(saved_nif)
+		saved_nif.quick_implant(H)
+
+/datum/species/protean/get_bodytype(var/mob/living/carbon/human/H)
+	if(H)
+		return H.impersonate_bodytype || ..()
+	return ..()
+
+/datum/species/protean/handle_post_spawn(var/mob/living/carbon/human/H)
+	..()
+	H.synth_color = TRUE
+
+/datum/species/protean/equip_survival_gear(var/mob/living/carbon/human/H)
+	var/obj/item/stack/material/steel/metal_stack = new()
+	metal_stack.amount = 3
+
+	var/obj/item/clothing/accessory/permit/nanotech/permit = new()
+	permit.set_name(H.real_name)
+
+	if(H.backbag == 1) //Somewhat misleading, 1 == no bag (not boolean)
+		H.equip_to_slot_or_del(permit, slot_l_hand)
+		H.equip_to_slot_or_del(metal_stack, slot_r_hand)		
+	else
+		H.equip_to_slot_or_del(permit, slot_in_backpack)
+		H.equip_to_slot_or_del(metal_stack, slot_in_backpack)
+
+	spawn(0) //Let their real nif load if they have one
+		if(!H.nif)
+			var/obj/item/device/nif/bioadap/new_nif = new()
+			new_nif.quick_implant(H)
+		else
+			H.nif.durability = rand(21,25)
+
+/datum/species/protean/hug(var/mob/living/carbon/human/H, var/mob/living/target)
+	return ..() //Wut
+
+/datum/species/protean/get_blood_colour(var/mob/living/carbon/human/H)
+	return rgb(80,80,80,230)
+
+/datum/species/protean/get_flesh_colour(var/mob/living/carbon/human/H)
+	return rgb(80,80,80,230)
+
+/datum/species/protean/handle_death(var/mob/living/carbon/human/H)
+	to_chat(H,"<span class='warning'>You died as a Protean. Please sit out of the round for at least 30 minutes before respawning, to represent the time it would take to ship a new-you to the station.</span>")
```

Comment:
Whoa. I didn't know this was a thing and should probably add something similar to chimeras.